realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -1657,14 +1657,24 @@ public static void migrateRealm(RealmConfiguration configuration, @Nullable Real
     }
 
     /**
-     * Deletes the Realm file specified by the given {@link RealmConfiguration} from the filesystem.
+     * Deletes the Realm file along with the related temporary files specified by the given {@link RealmConfiguration}
+     * from the filesystem. Temporary file with ".lock" extension won't be deleted.
+     * <p>
      * All Realm instances must be closed before calling this method.
+     * <p>
+     * This doesn't support deleting synced Realm for now.
      *
      * @param configuration a {@link RealmConfiguration}.
-     * @return {@code false} if a file could not be deleted. The failing file will be logged.
+     * @return {@code false} if the realm file could not be deleted. Temporary files deletion failure won't impact on
+     * the return value. All of the failing file will be logged.
```

Comment:
Not sure I understand `All of the failing file will be logged` ?

Reply:
will change it to `all the failing file deletion`